pandas/io/formats/style.py

```diff
@@ -1512,26 +1586,24 @@ def _background_gradient(
         text_color_threshold: float = 0.408,
         vmin: Optional[float] = None,
         vmax: Optional[float] = None,
+        gmap: Optional[Union[Sequence, np.ndarray, FrameOrSeries]] = None,
+        axis_: Optional[Axis] = None,
```

Comment:
why is this axis_?

Reply:
when you call `apply()`, `axis` is one of the native arguments so to pass the value of `axis` along with the applied function (for arg validation) you need a different kwarg name, i.e. `apply(f, axis=axis, axis_=axis)`

Reply:
OK, I changed it to just `partial` it out: `apply(partial(f, axis=axis), axis=axis)`